### PR TITLE
Feature/40010

### PR DESCRIPTION
**Pull request by UrvashiChaudhary**
- Create a file Database.ts inside libs folder.
- Install npm package mongoose and @types/mongoose.
- Add MONGO_URL=mongodb://localhost:27017/express-training to your .env file.
- Create a class named Database inside Database.ts
- It will contain two methods(open and disconnect)
- open This method will accept mongoURL as argument and will establish a connection with mongodb.
- disconnect This method will close the connection with database.
- Use the method open to establish connection inside run method of class Server and only after successfull connection with 
- mongo app will start listening.

### CODE REVIEW

**Comment by adilsuccessive** [comment]
Resolve conflicts

**Comment by UrvashiChaudhary** [comment]
> Resolve conflicts

resolved

File: src/libs/validationHandler.ts [hunk 1]
```diff
@@ -64,11 +62,15 @@ export default (config) => (req, res, next) => {
             }
         }
         // Checking for default
-        if (obj.default) {
+        /* if (obj.default) {
             if (isNull(values[0])) {
+<<<<<<< HEAD
```
Comment (adilsuccessive) [inline src/libs/validationHandler.ts]: Resolve this conflict

**Review by adilsuccessive — CHANGES_REQUESTED** [review]

File: src/libs/validationHandler.ts (hunk 1 above)
Comment (UrvashiChaudhary) [reply src/libs/validationHandler.ts]: resolved

Comment (UrvashiChaudhary) [reply src/libs/validationHandler.ts]: resolved

**Pull request merged**